library/lua/custom-raw-tokens.lua

```diff
@@ -0,0 +1,243 @@
+--[[
+custom-raw-s
+Allows for reading custom tokens added to raws by mods
+by Tachytaenius (wolfboyft)
+
+Yes, custom raw tokens do quietly print errors into the error log but the error log gets filled with garbage anyway
+]]
+
+local _ENV = mkmodule("custom-raw-tokens")
+
+local eventful = require("plugins.eventful")
```

Comment:
@lethosor what do you think about this? is it acceptable to depend on a plugin in a core library? should `eventful` (and other "core-like" plugins like `debug`) be moved into the core lib to avoid the architecture inversion?

alternately, we could use `sc-script` to call a script (or this module directly?) with specific parameters (e.g. `--reset`) on UNLOAD events. that just uses core functionality instead of `eventful`, but is a little convoluted for this simple use case.